app/views/sufia/batch_uploads/_form_files.html.erb

```diff
@@ -74,7 +74,19 @@
 </script>
 <!-- The template to display files available for download -->
 <script id="template-download" type="text/x-tmpl">
-{% for (var i=0, file; file=o.files[i]; i++) { %}
+{%
+
+    // I am not happy with this global declaration.
+    // Using "var setAllResourceTypes" or "function setAllResourceTypes()" does not work. Not sure why.
+    // Perhaps this kind of declaration is not a problem due to the way the JavaScript is executed???
+    setAllResourceTypes = function() {
```

Comment:
I am not sure why neither `function setAllResourceTypes() {...}` nor `var setAllResourceTypes = function() {...}` worked. Not a show stopper but I thought I would highlight this in case any one has ideas on why this is the case.